src/autoimport/services.py

```diff
@@ -4,12 +4,58 @@
 and handlers to achieve the program's purpose.
 """
 
-from typing import Any, Dict, Optional, Tuple
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple
 
 from _io import TextIOWrapper
+from maison.config import ProjectConfig
 
 from autoimport.model import SourceCode
 
+# To consider: use `xdg.xdg_config_home() / "autoimport" / "config.toml"`
+GLOBAL_CONFIG_PATH = "~/.config/autoimport/config.toml"
+CONFIG_PROJECT_NAME = "autoimport"
```

Comment:
I agree with you that using `xdg.xdg_config_home() / "autoimport" / "config.toml"` is a better solution than hardcoding `~/.config`